### ISSUE TITLE
Radio Button Support

### ISSUE THREAD

**Issue by zans-laksa** [scandipwa/scandipwa#3039]
We have a `Field` component that supports various types and `fieldMap` from in forms where we can pass a config to render fields.
We should check whether SPWA correctly handles radio buttons:
- Check if radio buttons fit into the design
- Check if radio buttons clickable
- Check if radio buttons are grouped by name
- Check radio button validation
- Check whether correct value of radio button is submitted
- Check how radio buttons handles values set programmatically

**Comment by aleksandrakorolova** [comment]
Need to test for config product after #3447 fix
Created separate task for found issues #3459
